File: Week5/Mini Projects/ObjectiveCPrac/Student.m
//
//  Student.m
//  ObjectiveCPractice
//
//  Created by Kaleo Kim on 3/3/16.
//  Copyright © 2016 Kaleo Kim. All rights reserved.
//

#import "Student.h"

@implementation Student

- (instancetype)initWithName: (NSString *)name age:(NSString *)age; {
    
    self = [super init];
    if(self) {
        _name = name;
        _age = age;
    }
    
    return self;
}

@end

File: Week1/StudentList/StudentList/Student.swift
//
//  Student.swift
//  StudentList
//
//  Created by Kaleo Kim on 2/4/16.
//  Copyright © 2016 Kaleo Kim. All rights reserved.
//

import UIKit

class Student: NSObject {

    let name: String
    let phoneNumber: String
    let hometown: String
    
    init(name: String, phoneNumber: String, homeTown: String) {
        self.name = name
        self.phoneNumber = phoneNumber
        self.hometown = homeTown
    }
    
    
}
